### DISCUSSION TITLE
4th KRATOS COMMUNITY MEETING

### DISCUSSION THREAD

**Discussion by maceligueta** [KratosMultiphysics/Kratos#8723] [General]
# **4th KRATOS COMMUNITY MEETING**

### https://unipd.zoom.us/j/81286277636  
[****** add meeting to my Google Calendar ******](https://calendar.google.com/event?action=TEMPLATE&tmeid=MGw5bTdzMG50bmV2bTlpYTIxMnM3aHBiZWYgNW9kY2liY2FsYmY5bWQ1dDZlcTc5OW41cXNAZw&tmsrc=5odcibcalbf9md5t6eq799n5qs%40group.calendar.google.com)

This meeting follows the series started with the [1st Kratos Community Meeting](https://github.com/KratosMultiphysics/Kratos/discussions/8239)

## ADDRESSED TOPICS:
The addressed topics will be proposed as answers to this post, and can be upvoted or supported by the Community.
If other questions do not receive more support, presumably we will address the following topics:

- Team Maintainers / Code Owners discussion
- Documentation

### •	Post your questions, doubts, issues and suggestions as contributions to this discussion before 21st May 2021

•	Use **‘thumb up’ or upvoting** to support previously written questions. Some of the questions are ported from the previous meeting, because there was not enough time to address all of them. The most popular questions will have priority

## AGENDA

▪	Welcome and introduction (5 mins)

▪	Questions are addressed (25 mins)

@maceligueta @antonialarese @rolandwuechner

**Comment by maceligueta** [comment]
Following #8695, we could discuss about the role of Team Maintainers and Code owners, and how to interact between teams when bugs are detected or features are requested, as well as the role of the developers with respect to pull requests.

**Comment by pooyan-dadvand** [comment]
I have another important topic to be discussed in the community meeting but I am not sure if it will fit in this one or we should schedule one especially for it:

**How can we accelerate our review process without sacrificing the quality of the code?**

I am asking here because I think application maintainers can help in reducing the load of the  @KratosMultiphysics/technical-committee especially for the main applications that have many derived ones.

**Comment by maceligueta** [comment]
Hi all, this is just a reminder that this short meeting will take place tomorrow morning!
